### ISSUE TITLE
Shuffle/Reverse events don't come with paths

### ISSUE THREAD

**Issue by hamptonsmith** [gullerya/object-observer#10]
When a nested array is `sort()`ed or `reverse()`d, the library generates a `shuffle` or `reverse` event, respectively.  These events do not include a path and it is therefore not possible to distinguish which internal array has been modified.  For example:

```javascript
> const Observe = require('object-observer');
> const target = Observe.from({foo: [1, 3, 2], bar: [7, 9, 8]});
> target.observe(c => { console.log(c); });
> target.foo.sort();
[ { type: 'shuffle' } ]
```

Ideally these events would come with a `path` field, just like other sorts of events:

```javascript
> target.foo.sort();
[ { type: 'shuffle', path: ['foo'] } ]
```

**Comment by gullerya** [comment]
Sounds right to me and shouldn't be a problem to achieve.
I'm on it.

**Comment by gullerya** [comment]
Available in version 1.1.0.
